### DISCUSSION TITLE
Non-daily data

### DISCUSSION THREAD

**Discussion by chrissype** [ourownstory/neural_prophet#217] [Q&A - get help using NeuralProphet] [answered]
Any plans to implement this? I really like the rest of the repo and API but this seems like an overly restrictive criterion for a time series forecasting lib...

**Comment by ourownstory** [comment, marked as answer]
Hi @chrissype, We have been supporting non-daily data for a while now. Less recent versions had a few areas of code that were not sufficiently tested for non-daily data. That has changed though. The current code on master branch should fully support all data frequencies!
